index.html

```diff
@@ -5,11 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="main-header">
-      <a href=""><logo>Noel Bryant</logo></a>
-    </header>
+    <div class="headline-container">
+      <header>
```

Comment:
- [ ] The project requirement states that you should align both sections with Flexbox. The sections referred to here are the toolbar (header) and headline. Please double-check the[ project requirements.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)
![inst](https://github.com/NoelLincoln/mportfolio/assets/99656687/ee12dea5-8210-46d5-9350-a3ce700d2356)